.changeset/wet-readers-join.md

```diff
@@ -0,0 +1,30 @@
+---
+'astro': minor
+---
+
+Shipped a new SSR mode, called `split`. 
+When enabled, Astro will emit a file for each page, which will render one single page.
```

Comment:
```suggestion
When enabled, Astro will "split" the single `entry.mjs` file and instead emit a separate file to render each individual page during the build process.
```
I'm assuming this is where the name "split" came from?  :smile: It would be good to have something to connect this name to what it does.